src/main.cpp

```diff
@@ -2063,13 +2064,14 @@ static int64_t nTimePostConnect = 0;
  * corresponding to pindexNew, to bypass loading it again from disk.
  */
 bool static ConnectTip(CValidationState &state, CBlockIndex *pindexNew, const CBlock *pblock) {
+    const CChainParams& chainparams = Params();
```

Comment:
Nit,  why did you opt for `chainparams` over `chainParams`?  camelCase is used consistently for most other variables.